intellij-plugin-structure/structure-edu/src/main/kotlin/com/jetbrains/plugin/structure/edu/EduPlugin.kt

```diff
@@ -19,12 +22,61 @@ data class EduPlugin(
   val language: String? = null,
   val programmingLanguage: String? = null,
   val eduPluginVersion: String?,
-  val items: List<String> = emptyList()
+  val eduStat: EduStat? = null
 
 ) : Plugin {
   override val pluginVersion: String? = null
   override val url: String = ""
   override val changeNotes: String? = null
 
   val parsedEduVersion = EduPluginVersion.parse(eduPluginVersion)
-}
\ No newline at end of file
+}
+
+data class EduStat(
+  val sections: List<String>,
+  val lessons: List<String>,
+  val tasks: Map<String, Int>
+) {
+
+  fun countInteractiveChallenges(): Int {
```

Comment:
as I understood there are methods for tests. Let's put in test module

Reply:
I've added them not for tests but for the easy access to the information needed in the additional information section.
![image (1)](https://user-images.githubusercontent.com/437080/91668668-10fcce80-eb17-11ea-8420-8c766e0036d7.png)

Reply:
I will do it on the Marketplace side